lib/Listener/PostLoginListener.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
```suggestion
<?php

declare(strict_types=1);

```

and copyright header

Reply:
Done